src/Fixer/Casing/LowercaseConstantsFixer.php

```diff
@@ -24,8 +27,20 @@
  *
  * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
  */
-final class LowercaseConstantsFixer extends AbstractFixer
+final class LowercaseConstantsFixer extends AbstractFixer implements ConfigurationDefinitionFixerInterface
 {
+    private $fixCallback;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function configure(array $configuration = null)
+    {
+        parent::configure($configuration);
+
+        $this->resolveFixCallback();
```

Comment:
Why the extra method, and not just place the oneliner here?

Reply:
Good point. Will do.